src/index.ts

```diff
@@ -73,6 +74,12 @@ class Monika extends Command {
     // Check if config is valid
     const isConfigValid: Validation = await validateConfig(config)
     if (isConfigValid.valid) {
+      if (process.env.NODE_ENV !== 'test') {
```

Comment:
It would be better if we mock notificationChecker function in the test environment, it makes the code cleaner

Reply:
Currently this project doesn't have a mocking library. I've been created issue for this (#88).